lib/widgets/action_sheet.dart

```diff
@@ -402,3 +433,24 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MessageActionSheetCancelButton extends StatelessWidget {
+  const MessageActionSheetCancelButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
+    return ElevatedButton(
+      style: ElevatedButton.styleFrom(
+        elevation: 0,
+        backgroundColor: designVariables.actionSheetCancelButtonBackground,
+        foregroundColor: designVariables.actionSheetCancelButtonForeground,
+        shadowColor: Colors.transparent,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(7)),
+      ),
```

Comment:
Can use `TextButton` here:

```suggestion
    return TextButton(
      style: TextButton.styleFrom(
        backgroundColor: designVariables.actionSheetCancelButtonBackground,
        foregroundColor: designVariables.actionSheetCancelButtonForeground,
        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(7)),
      ),
```

Reply:
Also add explicit padding of `EdgeInsets.all(10)` as per Figma design:

<img width="524" alt="Screenshot 2024-08-06 at 21 50 27" src="https://github.com/user-attachments/assets/0ae4408c-924c-4380-aae9-9177a601078c">